src/components/FilterPanel/FilterPanel.docs.mdx

```diff
@@ -0,0 +1,42 @@
+
+import { FilterPanel } from './FilterPanel';
+import { DEFAULT_FILTERS } from './FilterPanel.constants';
+
+# FilterPanel
+
+El componente `FilterPanel` permite seleccionar un filtro entre varias opciones, mostrando el estado seleccionado y disparando un callback al cambiar.
```

Comment:
Documentation text is written in Spanish. The repository guidelines require English as the standard language for code, comments and docs.

**Sugerencia:**
```typescript
Translate the doc content to English. Example:

The `FilterPanel` component allows selecting a filter option, displays the selected state and triggers a callback when the selection changes.
```